src/frontend/src/components/authenticateBox.ts

```diff
@@ -45,11 +48,12 @@ export type AuthnTemplates = {
  * to II or to another dapp */
 export const authenticateBox = async (
   connection: Connection,
+  i18n: I18n,
```

Comment:
@peterpeterparker might ask you to use an object instead

Reply:
i tried to changes as little as possible.